packages/page-accounts/src/modals/CreateConfirmation.tsx

```diff
@@ -10,10 +10,10 @@ import { isHex } from '@polkadot/util';
 import { useTranslation } from '../translate';
 
 interface Props {
-  address: string;
+  address?: string;
```

Comment:
Why won't the address and name be available? If it is not, it means that we have done something wrong and should not be in here. So it really _should_ always require it.

Reply:
So the thing is I'm keeping the `AddressRow` on top of the Modal on each of the three steps, confirmation included. This makes more sense to me, as I have one place to configure its props and looks. but `CreateConfirmation` is currently used in `Derive` as well. So I wanted to keep it showing the `AddressRow` there, but hide it here -- so effectively optional `address` is just a way to hide the `AddressRow`. Maybe it makes more sense to change `Derive` in this PR as well in similar fashion.